packages/meditrak-app/app/sync/SyncPage.js

```diff
@@ -89,7 +90,7 @@ const localStyles = StyleSheet.create({
 });
 
 SyncPage.navigationOptions = {
-  headerBackImage: require('../images/x.png'),
+  headerLeft: <HeaderLeftButton source={require('../images/x.png')}></HeaderLeftButton>,
```

Comment:
headerBackImage no longer works... At least for custom HeaderBackButton that we import from `react-navigation`. So this is a work around